src/components/events/EventDisplay/index.tsx

```diff
@@ -8,7 +8,7 @@ interface EventDisplayProps {
 }
 
 const EventDisplay = ({ events, attendances }: EventDisplayProps) => {
-  const displayedEvents = events;
+  const displayedEvents = events.slice(0, Math.min(20, events.length));
```

Comment:
```suggestion
  const displayedEvents = events.slice(0, 20));
```

Does this work the same? I think it would do the same thing and just return the full array if it's shorter than 20. Also magic variable for 20 - I would rename MAX_VIEW_EVENTS or smth.